lib/IRGen/IRGenSIL.cpp

```diff
@@ -884,6 +884,9 @@ class IRGenSILFunction :
   void visitValueMetatypeInst(ValueMetatypeInst *i);
   void visitExistentialMetatypeInst(ExistentialMetatypeInst *i);
   void visitTupleExtractInst(TupleExtractInst *i);
+  void visitDestructureValueInst(DestructureValueInst *i) {
+    llvm_unreachable("unimplemented");
```

Comment:
I assume you're planning to finish implementing this before committing?  Or will this be a follow-up?

Reply:
I was going to lower it in the ownership eliminator in a follow on commit.